pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -1773,11 +1802,11 @@ def pygsti_seq(qb_names, pygsti_listOfExperiments, operation_dict,
         return seq_name
 
 
-def ro_dynamic_phase_seq(qbp_name, qbr_names,
-                         phases, operation_dict,
+def ro_dynamic_phase_seq(hard_sweep_dict, qbp_name, qbr_names,
```

Comment:
The updates to this function need some cleaning up. Or we just reject the changes for now.

Reply:
Completely removed from master. Should be put into master when it is cleaned up.